src/components/page/sectionBreak.jsx

```diff
@@ -29,10 +30,11 @@ class SectionBreak extends React.PureComponent {
   }
 }
 
-export default SectionBreak;
+export default withTranslation()(SectionBreak);
```

Comment:
`withTranslation` is a HOC that is used when we need to use the `t()` method to translate (when a prop can't be handed a `Trans` object)